src/main/java/com/google/cloud/spanner/pgadapter/statements/SessionStatementParser.java

```diff
@@ -285,12 +285,23 @@ static SetStatement parseSetStatement(SimpleParser parser) {
           ErrorCode.INVALID_ARGUMENT,
           "Invalid SET statement: " + parser.getSql() + ". Expected configuration parameter name.");
     }
-    builder.setName(name);
-
-    if (!(parser.eatKeyword("to") || parser.eatToken("="))) {
-      throw SpannerExceptionFactory.newSpannerException(
-          ErrorCode.INVALID_ARGUMENT,
-          "Invalid SET statement: " + parser.getSql() + ". Expected TO or =.");
+    else if(name.toString().equalsIgnoreCase("TIME")){
```

Comment:
Instead of this, it would be better to add a `if (parser.eat("time", "zone"))` before the statement that reads the variable name (the `parser.readTableOrIndexName()` statement on line 282). If you do that, then you can enter a separate branch that specifically handles this type of statement, instead of having to mix it with the normal `set ...` statements.